source/hardware/controllers/pioneer_ddj_sb3.rst

```diff
@@ -0,0 +1,153 @@
+
+===================== ================
+Output Channels       Assigned to
+===================== ================
+1-2                   Master
+3-4                   Headphones
+===================== ================
+
+This controller mapping does not support any inputs. The controller itself has a microphone input, but that cannot be controlled via MIDI.
+
+Controller Mapping
+------------------
+
+A schematic drawing with the control numbers that are used here can be found on the specified page in the User Manual in the Links section.
+
+.. _pioneer-ddj-SB3-decks:
+
+Deck section (p. 6)
+~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_sb3_decks.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (deck section)
+   :figclass: pretty-figures
+
+   Deck section (p. 6)
+
+===  =================================================  ============================================================================================
+No.  Control                                            Function
+===  =================================================  ============================================================================================
+1    :hwlabel:`TEMPO` slider                            Adjust track playing speed (can be adjusted via :hwlabel:`SHIFT` + :hwlabel:`BEAT SYNC`)
+2    Jog Wheel (top)                                    Pitch bend (nudge)
+2    :hwlabel:`VINYL` + Jog Wheel (top)                 Scratch (move play position)
+2    Jog Wheel (outer)                                  Pitch bend (nudge)
+2    :hwlabel:`SHIFT` + Jog Wheel (top)                 Fast-forward or fast reverse track
+3    :hwlabel:`HOT CUE`                                 Set hot cue mode
+3    :hwlabel:`SHIFT` + :hwlabel:`HOT CUE`              Set BEAT JUMP mode
+4    :hwlabel:`FX FADE`                                 Set FX fade mode
+4    :hwlabel:`SHIFT` + :hwlabel:`FX FADE`              Set roll mode
+5    :hwlabel:`PAD SCRATCH`                             Set pad scratch mode
+5    :hwlabel:`SHIFT` + :hwlabel:`PAD SCRATCH`          Set slicer mode
+6    :hwlabel:`SAMPLER`                                 Set sampler mode
+6    :hwlabel:`SHIFT` + :hwlabel:`SAMPLER`              Set trans mode
+7    :hwlabel:`VINYL`                                   Toggle vinyl mode
+7    :hwlabel:`SHIFT` + :hwlabel:`VINYL`                Toggle slip mode
+8    :hwlabel:`AUTO LOOP`                               Toggle auto loop
+8    :hwlabel:`SHIFT` + :hwlabel:`AUTO LOOP`            Cancel loop or reloop
+9    :hwlabel:`2X`                                      Double loop length
+10   :hwlabel:`1/2X`                                    Halve loop length
+11   Pad 1 - 8                                          Perform different functions depending on current mode
+12   :hwlabel:`PLAY/PAUSE` button                       Play / pause
+12   :hwlabel:`SHIFT` + :hwlabel:`PLAY/PAUSE` button    Return to temporary cue point
```

Comment:
I think this function should be on SHIFT + CUE instead, then you could use SHIFT + PLAY for censoring.

Reply:
These are the original (OOB) mappings for the controller.